addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -474,6 +474,11 @@ export class TablePlugin extends Plugin {
 
     hanldeFirefoxSelection(ev = null) {
         const selection = this.document.getSelection();
+        const anchorNode =
```

Comment:
I guess it would be better to use the function `isSelectionInEditable` in the `selection_plugin` so that we have only one source of truth about if the selection is in the editable or not. You can add  the function in the shared functions of the plugin and you can use it in the table plugin that way